cpplib/include/easy/cache.h

```diff
@@ -16,9 +16,10 @@ class Cache {
     using wrapper_ty = decltype(easy::jit(std::forward<T>(Fun), std::forward<Args>(args)...));
 
     void* FunPtr = reinterpret_cast<void*>(meta::get_as_pointer(Fun));
-    auto CacheEntry = Cache_.insert(std::make_pair(Key(FunPtr,
-                                                       get_context_for<T, Args...>(std::forward<Args>(args)...)),
-                                                   FunctionWrapperBase()));
+    auto CacheEntry =
+        Cache_.insert(std::make_pair(Key(FunPtr,
+                      get_context_for<T, Args...>(std::forward<Args>(args)...)),
+                                     FunctionWrapperBase()));
```

Comment:
Could be emplace instead of insert + make_pair

Reply:
Done!